xwiki-platform-core/xwiki-platform-appwithinminutes/xwiki-platform-appwithinminutes-ui/src/main/resources/AppWithinMinutes/ClassEditSheet.xml

```diff
@@ -614,7 +614,9 @@ xcontext.put('propertyCustomDisplayer', new PropertyCustomDisplayer(xcontext))
       #set($queryString = 'wizard=true')
       #if(!$xwiki.exists($homePageRef))
         #set($homePageTitle = '$services.localization.render(''platform.appwithinminutes.appHomePageTitle'', [$doc.space])')
-        #set($queryString = "$queryString&amp;editor=inline&amp;template=AppWithinMinutes.LiveTableTemplate&amp;AppWithinMinutes.LiveTableClass_0_class=$escapetool.url($doc.fullName)&amp;title=$escapetool.url($homePageTitle)&amp;parent=AppWithinMinutes.WebHome")
+        #set($wikiHomePageRef = $services.model.resolveDocument('', 'default'))
```

Comment:
Default = `$services.model.resolveDocument('', 'default')`, i.e. "the default homepage of a wiki" / "the default homepage of the default space of a wiki" / etc., according to the model's configured default entities.

Current = `$services.wiki.currentWikiDescriptor.mainPageReference`, i.e. the currently configured homepage of the current wiki. See http://extensions.xwiki.org/xwiki/bin/view/Extension/Wiki+Application#HConfigureWiki

Using the default might not make sense if the homepage is configured to be something else in the current instance.

At the same time, using the current homepage might not make sense when exporting the app and installing it on a vanilla environment.